Tests/TuistGeneratorTests/Linter/GraphLinterTests.swift

```diff
@@ -1080,6 +1093,75 @@ final class GraphLinterTests: TuistUnitTestCase {
         XCTAssertEqual(result, [])
     }
 
+    func test_lint_doesNotFlagDependenciesWithLessConfigurations() throws {
+        // Lower level dependencies could be shared by projects in the same workspace as such
+        // it is ok if a project only has some configurations as long as it is a subset of what the dependency has
```

Comment:
Would you mind rehrasing this a little bit, I find it's a bit difficult to understand what you mean by this.

We can also include a link to this PR for future reference.

Reply:
Updated the comment. I also found it confusing, I initially just copied it from the previous test with some amendments for consistency.